checker/src/org/checkerframework/checker/index/IndexChecker.java

```diff
@@ -1,22 +1,20 @@
 package org.checkerframework.checker.index;
 
-import org.checkerframework.checker.index.lowerbound.LowerBoundChecker;
+import org.checkerframework.checker.index.upperbound.UpperBoundChecker;
 
 /**
  * A type checker for preventing out-of-bounds accesses on arrays. Contains two subcheckers that do
  * all of the actual work: the Lower Bound Checker and the Upper Bound Checker.
  *
  * @checker_framework.manual #index-checker Index Checker
  */
-public class IndexChecker extends LowerBoundChecker {
+public class IndexChecker extends UpperBoundChecker {
     // Ideally, the Index Checker would be an AggregateChecker that ran the Lower and Upper Bound
```

Comment:
Just delete this whole comment.  Instead, you can just say something about how the Upper Bound Checker runs the other checkers and then reference https://github.com/typetools/checker-framework/blob/master/checker/src/org/checkerframework/checker/index/Checker-dependencies.txt.  (And that file needs to be updated now.)